library_generation/test/resources/test-owlbot/fixtures/java_templates/render-readme/README-golden.md

```diff
@@ -0,0 +1,202 @@
+# Google Cloud Asset Inventory Client for Java
+
+Java idiomatic client for [Cloud Asset Inventory][product-docs].
+
+[![Maven][maven-version-image]][maven-version-link]
+![Stability][stability-image]
+
+- [Product Documentation][product-docs]
+- [Client Library Documentation][javadocs]
+
+
+:bus: In October 2022, this library has moved to
+[google-cloud-java/java-asset](
+https://github.com/googleapis/google-cloud-java/tree/main/java-asset).
+This repository will be archived in the future.
+Future releases will appear in the new repository (https://github.com/googleapis/google-cloud-java/releases).
+The Maven artifact coordinates (`com.google.cloud:google-cloud-asset`) remain the same.
+
+## Quickstart
+
+
+If you are using Maven, add this to your pom.xml file:
+
+
+```xml
+<dependency>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-asset</artifactId>
+  <version>None</version>
```

Comment:
I guess the version being `None` is because we don't have a versions.txt supplied?

Reply:
The version is passed in from env variable, I've added the env variable in the test so that the version is rendered.